src/bin/lfortran.cpp

```diff
@@ -1400,6 +1400,35 @@ int compile_to_object_file_c(const std::string &infile,
     return 0;
 }
 
+int runCommandAndGetStdout(const std::string& command, std::string &stdoutOutput) {
```

Comment:
To be consistent with the rest of the file:
```suggestion
int run_command_and_get_stdout(const std::string& command, std::string &stdoutOutput) {
```